zebrad/src/commands/start.rs

```diff
@@ -65,7 +67,8 @@ impl StartCmd {
         .await;
 
         info!("initializing mempool");
-        let mempool = mempool::Mempool::new(config.network.network);
+        let mempool_service = BoxService::new(Mempool::new(config.network.network));
+        let mempool = ServiceBuilder::new().buffer(20).service(mempool_service);
```

Comment:
The state seems to do the same but do you know why a buffer of `20` is the right thing to do here ?

Reply:
Good question, I just copied the value used in state. I'm not sure how to decide on the number. (Maybe @dconnolly can help?)

Reply:
We don't need to do this now and as the state is using this buffer size too the question is a bit off from this PR. I moved it to a new ticket https://github.com/ZcashFoundation/zebra/issues/2743